src/tz390.java

```diff
@@ -6649,11 +6987,15 @@ public boolean fp_get_dfp_bin(int dfp_type,BigDecimal dfp_bd){
     	}
     	return false;
     }
-    private long get_dfp_ccf_digits(int tot_digits,int digit_offset, int digit_count){
-    	/*
-    	 * return long with 1 to 6 DPD densly packed deciaml
-    	 * truples of 10 bits representing 3 digits.
-    	 */
+
+
+
+    /**
+     * return long with 1 to 6 DPD densly packed deciaml
+     * truples of 10 bits representing 3 digits.
+     */
+private long get_dfp_ccf_digits(int tot_digits,int digit_offset, int digit_count){
```

Comment:
Add @param and @return values to javadoc.

Reply:
The routine refers t dfp_digits where I think it should refer to tot_digits. Will work okay as long as global variable dfp_digits is indeed intended to be the input field. Do we need to change this? Could you please have a look at it?